lib/bamboo/adapters/sendgrid_helper.ex

```diff
@@ -0,0 +1,77 @@
+defmodule Bamboo.SendgridHelper do
+  @moduledoc """
+  Functions for using features specific to Sendgrid template substitution tags.
```

Comment:
I think the moduledoc should be bit more generic. maybe `Functions for using feature specific to Sendgrid`?